### PR TITLE
configuration: Update scrape config

### PR DESCRIPTION
**Pull request by beorn7**
@sdurrheimer Now that we have started to update documentation: Need this documentation update, too.

### CODE REVIEW

**Comment by beorn7** [comment]
Ping! Or anybody... @fabxc ?

File: content/docs/operating/configuration.md [hunk 1]
```diff
@@ -220,13 +220,14 @@ A `tls_config` allows configuring TLS connections.
 ```
 
 
-### `<target_group>`
+### `<static_config>`
 
-A `target_group` allows specifying a list of targets and a common label set for them.
-They are the canonical way to specify static targets in a scrape configuration.
+A `static_confic` allows specifying a list of targets and a common label set
```
Comment (brian-brazil) [inline content/docs/operating/configuration.md]: We keep a note of the old name in the docs for a few months, to reduce the inevitable support hassles.

Comment (beorn7) [inline content/docs/operating/configuration.md]: Are you referring to documenting the backwards-compatibility, or do you want a note like "This option was called `target_group` up to 0.19?
In which case the same should be done for the file SD, no?

Comment (brian-brazil) [inline content/docs/operating/configuration.md]: The former, and yes that'd make sense.

Comment (beorn7) [inline content/docs/operating/configuration.md]: But we have no backwards-compatibility for the file SD.

Perhaps you mean "the latter"?

Comment (brian-brazil) [inline content/docs/operating/configuration.md]: Yes, I did mean the latter.

Comment (beorn7) [inline content/docs/operating/configuration.md]: Alright. :)
Updating now, so that we can then update the docs in master for 0.20.

**Comment by beorn7** [comment]
@brian-brazil Updated. PHAL.

File: content/docs/operating/configuration.md (hunk 1 above)
Comment (brian-brazil) [inline content/docs/operating/configuration.md]: Weren't you going to add a mention of the name pre-0.20?

**Comment by brian-brazil** [comment]
:+1:

File: content/docs/operating/configuration.md (hunk 1 above)
Comment (beorn7) [inline content/docs/operating/configuration.md]: Yes, but at a different place. `target_group` was never a keyword in the config (only `target_groups`).

**Pull request merged**